provision/provision-site.sh

```diff
@@ -76,6 +76,11 @@ function vvv_provision_site_nginx() {
   sed -i "s#{vvv_path_to_site}#${VM_DIR}#" "/etc/nginx/custom-sites/${DEST_NGINX_FILE}"
   sed -i "s#{vvv_site_name}#${SITE}#" "/etc/nginx/custom-sites/${DEST_NGINX_FILE}"
   sed -i "s#{vvv_hosts}#${VVV_HOSTS}#" "/etc/nginx/custom-sites/${DEST_NGINX_FILE}"
+
+  if [ 'php' != "${NGINX_UPSTREAM}" ] && [ ! -f "/etc/nginx/upstreams/${NGINX_UPSTREAM}.conf" ]; then
```

Comment:
I think this should do the trick:

```suggestion
  if [ ! -f "/etc/nginx/upstreams/${NGINX_UPSTREAM}.conf" ]; then
```